scripts/validate-env.sh

```
@@ -0,0 +1,370 @@
+
+# Function to validate database path
+validate_db_path() {
+    local db_path=$1
+    
+    if [ -z "$db_path" ]; then
+        add_error "DB_PATH is not set"
+        return 1
+    fi
+    
+    # Check if directory exists or can be created
+    local db_dir=$(dirname "$db_path")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)